src/SFA.DAS.PSRService.Web/Startup.cs

```diff
@@ -94,11 +96,28 @@ private IServiceProvider ConfigureIOC(IServiceCollection services)
                 config.For<SingleInstanceFactory>().Use<SingleInstanceFactory>(ctx => t => ctx.GetInstance(t));
                 config.For<MultiInstanceFactory>().Use<MultiInstanceFactory>(ctx => t => ctx.GetAllInstances(t));
                 config.For<IMediator>().Use<Mediator>();
+
+                AddDataProtectionSettings(services);
```

Comment:
I would move this out of ConfigureIOC and put it alongside the extensions

`services.AddHealthChecks()` etc